docs/ref/contrib/postgres/fields.txt

```diff
@@ -275,8 +275,8 @@ A more useful index is a ``GIN`` index, which you should create using a
 
 .. class:: HStoreField(**options)
 
-    A field for storing mappings of strings to strings. The Python data type
-    used is a ``dict``.
+    A field for storing mappings of strings to strings or nulls. The Python data type
+    used is a ``dict``. Database null values are represented as None in Python.
```

Comment:
Include a [versionchanged](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features) annotation and a mention in the release notes.